source/advanced_components/docker/docker_integration.rst

```diff
@@ -0,0 +1,158 @@
+===========================================
+Docker and OpenNebula integration
+===========================================
+
+For docker integration we need to install docker machine and docker at OpenNebula frontend:
+
+.. prompt:: bash # auto
+
+    # curl -L https://github.com/docker/machine/releases/download/v0.14.0/docker-machine-`uname -s`-`uname -m` >/tmp/docker-machine && sudo install /tmp/docker-machine /usr/local/bin/docker-machine
+    # sudo apt-get install docker-ce
+
+Official documentation for Docker Machine is available `here <https://docs.docker.com/machine/>`__.
+
+Usage
+================================================================================
+
+Once we have all the requirements we need to provide docker a user to use for docker-machine with OpenNebula, the user will need permissions to create / manage instances.
```

Comment:
Once we have all the requirements we need to provide an OpeNebula user to docker-machine